peekingduck/pipeline/nodes/output/media_writer.py

```diff
@@ -30,7 +30,25 @@
 
 
 class Node(AbstractNode):
-    """Node that processes videos and images as primary source input"""
+    """Node that processes videos and images as primary source input.
```

Comment:
can try reword this? I dont think its very representative of `media_writer`'s function

Reply:
I rewrote the description to better represent the node.